internal/cli/main.go

```diff
@@ -145,6 +163,11 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 		}
 
 	case "goto":
+		if flag.Arg(1) == helpArgument {
```

Comment:
Use a `flag.NewFlagSet()` to parse the the argument. Can you create create a new method that creates a FlagSet with `-help` argument parsing and use that for all of the sub-commands?

Reply:
I updated the PR